.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: python
 python:
  - "2.7"
-# - "3.3"
+ - "3.3"
```

Comment:
Should probably either change this to 3.6, or add more languages (fwiw django 2.0 will only support 3.5+ so 3.5 & 3.6 seem like reasonable choices)